shell/components/form/NameNsDescription.vue

```diff
@@ -194,50 +198,56 @@ export default {
     },
 
     namespaces() {
-      const inStore = this.$store.getters['currentStore'](this.namespaceType);
-      const choices = this.namespacesOverride || this.$store.getters[`${ inStore }/all`](this.namespaceType);
-
-      const out = sortBy(
-        choices.filter( this.namespaceFilter || ((choice) => {
-          const isSettingSystemNamespace = this.$store.getters['systemNamespaces'].includes(choice.metadata.name);
-
-          return this.isVirtualCluster ? !choice.isSystem && !choice.isFleetManaged && !isSettingSystemNamespace : true;
-        })).map(this.namespaceMapper || ((obj) => {
-          return {
-            label: obj.nameDisplay,
-            value: obj.id,
-          };
-        })),
-        'label'
-      );
+      const currentStore = this.$store.getters['currentStore'](this.namespaceType);
+      const namespaces = this.namespacesOverride || this.$store.getters[`${ currentStore }/all`](this.namespaceType);
+
+      const filtered = namespaces.filter( this.namespaceFilter || ((namespace) => {
+        const isSettingSystemNamespace = this.$store.getters['systemNamespaces'].includes(namespace.metadata.name);
+
+        if (this.isVirtualCluster) {
+          // For virtual clusters, filter out the namespace
+          // if it is a system namespace or if it is managed by
+          // Fleet.
+          return !namespace.isSystem && !namespace.isFleetManaged && !isSettingSystemNamespace;
+        }
+
+        // By default, include the namespace in the dropdown.
+        return true;
+      }));
+
+      const withLabels = filtered.map(this.namespaceMapper || ((obj) => {
+        return {
+          label: obj.nameDisplay,
+          value: obj.id,
+        };
+      }));
+
+      const sortedByLabel = sortBy(withLabels, 'label');
 
       if (this.forceNamespace) {
-        out.unshift({
+        sortedByLabel.unshift({
           label: this.forceNamespace,
           value: this.forceNamespace,
         });
       }
 
-      return out;
+      return [
+        {
+          label: this.t('namespace.createNamespace'),
+          value: ''
+        },
+        {
+          label:    'divider',
+          disabled: true,
+          kind:     'divider'
+        },
+        ...sortedByLabel
+      ];
     },
 
     isView() {
       return this.mode === _VIEW;
     },
-
-    colSpan() {
```

Comment:
I deleted this property because it was only used for the workload forms and I refactored the workload index so that it doesn't need it